deployment/hasura/migrations/AerieScheduler/1_optional_sim_per_goal/down.sql

```diff
@@ -0,0 +1,5 @@
+alter table scheduling_specification_goals drop column simulate_after;
+
+comment on column scheduling_specification_goals.simulate_after is null;
```

Comment:
This comment needs to be set to null before the column is dropped, otherwise an error is thrown when running the down.sql

Reply:
done